Add some information about how to test master

gsmet opened the pull request.

gsmet committed:
Add some information about how to test master

geoand commented:
Isn't this fairly similar to what we already have at https://github.com/quarkusio/quarkus/blob/82bbfae8ccece40815f5feded3bbcc6d1664c1f2/CONTRIBUTING.md#usage?

gsmet commented:
Ah yes, looks like it. But it's a bit buried and too chatty for my taste when someone just wants to quickly check master.

geoand commented:
Understood

gsmet commented:
I will tweak it a bit and add some links to the other section.

geoand commented:
Cool, let me know when you want a new review

gsmet committed:
Add some guidelines

This was discussed a long time ago and Emmanuel asked we put these in
writing in the CONTRIBUTING guide. It's a start, we can add more later.

gsmet commented:
@geoand I gave it another shot. Also added some guidelines as requested a while ago by Emmanuel.

geoand commented on CONTRIBUTING.md:
```diff
@@ -128,8 +174,23 @@ After the build was successful, the artifacts are available in your local Maven
 To include them into your project a few things have to be changed.
 
-#### With Gradle
+#### With Maven
```
I like how Maven if first now :)

geoand reviewed (approved):
Clarifies things quite a bit!

famod commented:
`-T` > 1 for Maven might increase developer joy. :wink:
